grails-app/controllers/au/org/emii/aatams/SensorController.groovy

```diff
@@ -32,8 +32,6 @@ class SensorController extends ReportController {
         def tag = tagFactoryService.lookupOrCreate(params.tag)
         assert(tag)
 
-        params.tag = tag
```

Comment:
Yeah, I agree it looks wrong. It's weird that it actually ran before.

I don't have time to test it locally.

Perhaps we could get @xhoenner to test it on the test or dev instances - paying particular attention to cross-referencing the id's between the tables?

It's an important fix - and deals directly with the integrity of the database - so it needs to be correct.